src/3d/chunks/qgschunkedentity_p.h

```diff
@@ -111,6 +113,20 @@ class QgsChunkedEntity : public Qt3DCore::QEntity
      */
     bool usingAditiveStrategy() const { return mAdditiveStrategy; }
 
+    /**
+     * Sets the limit of the GPU memory used to render the entity
+     * \since QGIS 3.24
+     */
+    void setGpuMemoryLimit( double gpuMemoryLimit ) { mGpuMemoryLimit = gpuMemoryLimit; }
+
+    /**
+     * Returns the limit of the GPU memory used to render the entity
+     * \since QGIS 3.24
+     */
+    double gpuMemoryLimit() const { return mGpuMemoryLimit; }
```

Comment:
is double appropriate here? Maybe add an explanation of the units involved here in the documentation